### DISCUSSION TITLE
equal angle plotting of GBPD

### DISCUSSION THREAD

**Discussion by alexcaustin** [mtex-toolbox/mtex#2185] [Ask Anything] [answered]
Hello, 
I'm trying to plot a full GBCD as equal angle plot. I know the default in MTex is equal area but I don't seem to be able to change this.
I have tried `plotGBPD(gbpd,cs,'projection','stereo')` and the `eangle` command but haven't had any luck

Thanks for your help,
Alex

**Comment by filippeof** [comment, marked as answer]
Hi Alex,
If you are using [this function](https://github.com/filippeof/phdThesis_Scripts/blob/main/Chapter7_GBPD-GBCDtoolbox/plotGBPD.m), try changing:
`        pcolor(equiGrid,idwVec,'projection','edist','upper','smooth'); %` 
with:
`        pcolor(equiGrid,idwVec,'projection','eangle','upper','smooth'); % `
Best,
Filippe
